cookiecutter.json

```diff
@@ -1,7 +1,7 @@
 {
 	"author_name": "Elixir Cloud AAI",
 	"author_email": "cloud-service@elixir-europe.org",
-	"development_status": "Alpha",
+	"development_status": "Beta",
```

Comment:
I think "Alpha" is more appropriate as a default. A project generally does not start in "beta" status (except probably this one after this PR is merged :rofl:).

Reply:
idunno, I was testing this package by creating a [test pypi release](https://test.pypi.org/project/python_cookiecutter/) and couldn't upload with `Alpha` as status.

Reply:
Where did you get this from in the first place? Shouldn't this be a [classifier?](https://pypi.org/classifiers/)? I don't think `development_status` is the standard way how to specify that.

And once we use a classifier, I think it should be `1 - Planning` or `2 - Pre-Alpha`.

Reply:
my bad, I didn't notice the number while testing, and tried to push with `4 - Alpha` which will obv fail, either way I have added all the classifier, and user can select which ever they want to generate the repo, some i think shouldn't have been added like `6 - Mature` to a cookiecutter, but for the sake of completeness, I;ve added all.

Reply:
default to `Planing`.